README.md

```diff
@@ -35,8 +36,10 @@ All of the options can be set after object creation with obvious setters:
 ### epub.setModified(modified)
 ### epub.setPublished(published)
 ### epub.setSource(source)
-### epub.setLanguage(language)
 ### epub.setDescription(description)
+### epub.setLanguage(language)
+
+Identifies the language used in the book content. The content has to comply with [RFC 3066](http://www.ietf.org/rfc/rfc3066.txt). [List of language codes](http://www.loc.gov/standards/iso639-2/php/code_list.php).
```

Comment:
Good point, I did notice some description on language after I wrote this. I will try to merge the 2.